src/styles/utils/_variables.scss

```
@@ -0,0 +1,13 @@
+$days-of-week:
+  mon 0,
+  tue 101px,
+  wed 202px,
+  thu 303px,
+  fri 404px,
+  sat 505px,
+  sun 606px;
+
+$last-days:
+  "28" 3,
+  "29" 2,
+  "30" 1;
```

And these can be numbers.
Also, you could use `from 28 to 31` instead of collection.
```suggestion
  28 3,
  29 2,
  30 1;
```